### DISCUSSION TITLE
PrestaShop 8.1.4 - Problem with the "disabled" modules on the "Module Manager" page

### DISCUSSION THREAD

**Discussion by Angar90** [PrestaShop/PrestaShop#35609] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Go to the "Module Manager" tab and disable one module.
After that the "Configure" button should be changed to "Enable", but Prestashop still displays the "Configure" button.

### Expected behavior

The "Configure" button should be changed to "Enable" like in the PrestaShop 1.7
Without this option is it hard to find which modules are disabled in the backoffice.

### Steps to reproduce

1.Go to the "Module Manager" tab.

2.Disable one module, for example the "Cross-selling" module:
https://prnt.sc/X4wVZC-uaLbq

3.After that the "Configure" text should be changed to "Enable" (in PrestaShop 1.7 it works correctly), but it still shows the text "Configure".

Screenshot from PrestaShop 8.1.4:
https://prnt.sc/xcrm_Bf--998

Screenshot from PrestaShop 1.7.8:
https://prnt.sc/QvuUMH3fKizV

Without this option is it hard to find which modules are disabled in the backoffice. I have this problem in all PrestaShop 8.x versions.

Also, sometimes I have a similar problem with the module's upgrade, after the upgrade, the old version number of the module is still displayed in the backoffice.

### PrestaShop version(s) where the bug happened

8.1.4

**Comment by matks** [comment]
Hello @Angar90 I think you wanted to report a bug so I transform this item in a bug report

**Comment by matks** [comment]
https://github.com/PrestaShop/PrestaShop/issues/35665
